test/support/api/forms.js

```diff
@@ -1,62 +1,54 @@
 import _ from 'underscore';
-import { v4 as uuid } from 'uuid';
 import { getResource, mergeJsonApi } from 'helpers/json-api';
 
-import fxTestForms from 'fixtures/test/forms';
+import fxTestForms from 'fixtures/collections/forms';
+
+const fxSampleForms = _.rest(fxTestForms, 2);
```

Comment:
since we're only using `fxTextForms[0]` this should be `_.rest(..., 1)`

Reply:
Yep, updated